Cargo.toml

```diff
@@ -45,6 +47,9 @@ pure-rust-locales = { version = "0.5.2", optional = true }
 wasm-bindgen = { version = "0.2", optional = true }
 js-sys = { version = "0.3", optional = true } # contains FFI bindings for the JS Date API
 
+[target.'cfg(windows)'.dependencies]
+winapi = { version = "0.3.0", features = ["std", "minwinbase", "minwindef", "timezoneapi"] }
+
```

Comment:
I suppose cargo's lack of target-specific optional dependencies means there's no good way to make this optional?

Reply:
I mean, does it work to just mark it optional? Did that, and that... seems to work ok!